pace-util/pace/util/quantity.py

```diff
@@ -392,12 +399,19 @@ def storage(self):
             )
         return self._storage
 
-    def _initialize_storage(self, data, origin, gt4py_backend: str):
+    def _initialize_storage(self, data, origin, gt4py_backend: str, mask: Tuple):
+        spatial_dims = [
+            i for j in [constants.X_DIMS, constants.Y_DIMS, constants.Z_DIMS] for i in j
+        ]
+        extra_dims = [i for i in self._dims if i not in spatial_dims]
```

Comment:
This code to compute extra_dims is duplicated. Instead of pulling it into a function, can you add a `pace.util.SPATIAL_DIMS` constant equal to `X_DIMS + Y_DIMS + Z_DIMS`, defined in the same file where these constants are defined? Then the one-liner for extra_dims is simple and fine to duplicate.

Reply:
Added `pace.util.SPATIAL_DIMS`